guide/contributing/codebase-overview.md

```diff
@@ -193,7 +197,7 @@ expect(myComponent.props().someBoolean).toBeTruthy()
 // => Expected false to be truthy.
 ```
 
-Always prefer "shallow" rendering, then "render", then "mount". Only "mount" if you are testing the lifecycle methods.
+Always prefer "shallow" rendering, then "render", then "mount". Only "mount" if you are testing the lifecycle methods. Using "render" is required for snapshot testing styled components.
```

Comment:
I would add a little explanation on why `render` is required for snapshots:

```
Using "render" is required for snapshot testing styled components because it prints all auto-generated CSS classes.
```